docs/en/advanced_tutorials/test_time_augmentation.md

```diff
@@ -0,0 +1,161 @@
+# Test time augmentation
+
+Test time augmentation (TTA) is a data augmentation strategy used during the testing phase. It involves applying various augmentations, such as flipping and scaling, to the same image and then merging the predictions of each augmented image to produce a more accurate prediction. To make it easier for user to use TTA, MMEngine provides [BaseTTAModel](mmengine.model.BaseTTAModel) class, which allows users to implement different TTA strategies by simply extending the `BaseTTAModel` class according to their needs.
+
+The core implementation of TTA is usually divided into two parts：
+
+1. Data augmentation：This part is implemented in MMCV, see the api docs [TestTimeAug](mmcv.transform.TestTimeAug) for more information.
+2. Merge the predictions：The subclasses of `BaseTTAModel.test_step` will merge the predictions of enhanced data to a more accurate prediction.
+
+## Get started
+
+A simple example of TTA is given in [example/test_time_augmentation.py](https://github.com/open-mmlab/mmengine/blob/main/examples/test_time_augmentation.py)
+
+### Prepare test time augmentation pipeline
+
+`BaseTTAModel` needs to be used with `TestTimeAug` implemented in MMCV:
+
+```python
+tta_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='TestTimeAug',
+        transforms=[
+            [dict(type='Resize', img_scale=(1333, 800), keep_ratio=True)],
+            [dict(type='RandomFlip', flip_ratio=0.),
+                dict(type='RandomFlip', flip_ratio=1.)],
+            [dict(type='PackXXXInputs', keys=['img'])],
+        ])
+]
+```
+
+It means that each resized image will be flipped, and enhanced into two images during test phase.
+
+### Define the merge strategy
```

Comment:
```suggestion
### Define the merging strategy
```

Reply:
"Merge strategy" is a reasonable verbal noun phrase.